README.md

```diff
@@ -23,17 +25,21 @@ Note: We recommend running these commands in a unix terminal. Windows users can
 
 1. Clone the repo.
     ```shell
-    git clone https://github.com/plaid/pattern-account-funding.git
-    cd pattern
+    git clone https://github.com/plaid/pattern-transfers.git
+    cd pattern-transfers
     ```
+1. Follow the instructions for [Transfer UI](https://plaid.com/docs/transfer/using-transfer-ui/#prerequisites-for-using-transfer-ui) to set up your link customization on the [Plaid developer dashboard](https://dashboard.plaid.com/team/api).
+
```

Comment:
The version of these instructions in the .env.template is not that helpful and should be updated (it doesn't tell you where in the transfer UI docs to look, I wasn't able to find the relevant paragraph). This version is better. 

But, I still think it would be even better thought if we could repeat the instructions here rather than linking people to another page -- being linked to another document in a README *feels* intimidating (even if the instructions on the other document are actually simple, as they are in this case) so if the instructions are short and simple, I think it would be better to provide them here and in .env.template, even if it's not very DRY

Reply:
done